tools/src/scie_pants/pants_version.py

```diff
@@ -99,6 +102,12 @@ def determine_tag_version(
     stable_version = Version(pants_version)
     if stable_version >= PANTS_PEX_GITHUB_RELEASE_VERSION:
         return ResolveInfo(stable_version, sha_version=None, find_links=None)
+    elif stable_version >= PANTS_GITHUB_WHEELS_START_VERSION:
```

Comment:
Just confirming my understanding: this is unlikely to be used for any _future_ versions of Pants, because they'll all use the PEX, right?

Reply:
That's the plan, yeah. Although they'll be uploaded to the cheeseshop for plugins support